sdk/program/src/vote/state/mod.rs

```diff
@@ -404,9 +404,29 @@ impl<I> CircBuf<I> {
     }
 }
 
+#[repr(C)]
+#[derive(Clone, Copy, Debug, Serialize, Deserialize, PartialEq, Eq)]
+#[cfg_attr(test, derive(Arbitrary, Default))]
+#[cfg_attr(feature = "frozen-abi", derive(AbiExample))]
+pub struct EpochCreditsItem {
```

Comment:
For this and the other item type too, we should add tests that check that
mem::size_of::<I>() == what_we_expect. With the current layout there's no
padding added by the compiler, and it's unlikely that any of the types used for
the fields will change, but tests are cheap and it's better to be extra careful
for stuff like this.